app/views/static_pages/cookies.html.erb

```diff
@@ -139,9 +139,9 @@
       </thead>
       <tbody>
         <tr class="govuk-table__row">
-          <td class="govuk-table__cell">XXXX</td>
+          <td class="govuk-table__cell">seen_cookie_banner</td>
           <td class="govuk-table__cell">Saves a message to let us know that you have seen our cookie message</td>
-          <td class="govuk-table__cell">After 1 month</td>
+          <td class="govuk-table__cell">After 1 year</td>
```

Comment:
I don't think this is a "welcome message", either. Maybe the text above just needs a bit of amendment.

Reply:
Amended this section to now refer to cookie banner.